Use recipe.yaml with pixi and rattler-build

rxm7706 opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->
Converted to recipe.yaml, using pixi and rattler-build that reduces build time from 32 Mins to 9 Mins
Added PiP check for all packages
Remove pip check for Ibis-impala - failing on thrift version 
Bump Version 10.5.0

rxm7706 committed:
Create recipe.yaml

rxm7706 committed:
Update conda-forge.yml

rxm7706 committed:
Delete recipe/meta.yaml

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

Commit:
MNT: Re-rendered with conda-build 25.3.2, conda-smithy 3.47.1, and conda-forge-pinning 2025.04.18.15.09.35

rxm7706 committed:
Remove pip check for Ibis-impala

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/recipe.yaml**:

 * ℹ️ `noarch: python` recipes should usually follow the syntax in our [documentation](https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python) for specifying the Python version.
   - For the `test.requires` section of `ibis-impala` output, you should usually use `python ${{ python_min }}.*` for the `python` entry.
   - If the package requires a newer Python version than the currently supported minimum version on `conda-forge`, you can override the `python_min` variable by adding a Jinja2 `set` statement at the top of your recipe (or using an equivalent `context` variable for v1 recipes).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14544247453](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14544247453). Examine the logs at this URL for more detail.</sub>

rxm7706 committed:
Remove incorrect thrift pin for ibis-impyla

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14543929560](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14543929560). Examine the logs at this URL for more detail.</sub>

rxm7706 commented:
@cpcloud @gforsyth
This is ready to review. Thank you and regards
Changes made
1. Converted to recipe.yaml, using pixi and rattler-build - reduces build time from 32 Mins to 9 Mins
2. Added PiP check for all packages
3. Remove pip check for Ibis-impala - failing on thrift version

rxm7706 committed:
Bump Version 10.5.0

rxm7706 commented:
@conda-forge-admin, please rerender

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-webservice.

I tried to rerender for you, but it looks like there was nothing to do.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/14544250429](https://github.com/conda-forge/conda-forge-webservices/actions/runs/14544250429). Examine the logs at this URL for more detail.</sub>
